app/views/components/_spinner.html.haml

```diff
@@ -1 +1 @@
-%img.spinner{ src: image_path("spinning-circles.svg"), style: "max-width: 100%" }
+%img.spinner{ src: image_path("spinning-circles.svg"), style: "max-width: 100%" }
```

Comment:
I guess the only modification here is that the newline at the end of the file has been deleted. 
We used to keep those new lines on each file. I'd vote to put new line on this file too ;)